src/services/art_generator.py

```
@@ -0,0 +1,197 @@
+    # Draw base shape
+    center_x, center_y = width // 2, height // 2
+    if "circular" in layout["base_shape"].lower():
+        draw_circular_shape(
+            art, center_x, center_y, color_map["primary"], elements["main_body"]
+        )
+    elif "tall" in layout["base_shape"].lower():
+        draw_tall_shape(
+            art, center_x, center_y, color_map["primary"], elements["main_body"]
+        )
+    else:
+        draw_default_shape(
```

Ruff (0.8.0)

Undefined name `draw_default_shape`

(F821)